src/test/language-server-test.ts

```diff
@@ -1399,6 +1399,34 @@ var Node: {\n\
 				});
 			});
 		});
+		describe('completions', function () {
+			before(function (done: () => void) {
+				utils.setUp(newLanguageHandler(), {
+					'a.ts': `class A {
```

Comment:
Can you add a test case for when the completion is defined in a separate file (i.e., imported)?